lfpykernels/kernel_approximator.py

```diff
@@ -386,6 +392,11 @@ def get_kernel(self, probes, Vrest=-65, dt=2**-4,
         H_YX: dict of ndarray
             shape (n_channels, 2 * tau // dt + 1) linear response kernel
         '''
+
+        if (not self.conductance_based) and g_eff:
```

Comment:
looks like this could be an `assert (not self.conductance_based) and g_eff, 'message'`